pkg/components/util/helm.go

```diff
@@ -150,6 +144,25 @@ func filterOutUnusedFiles(files map[string]string) map[string]string {
 	return ret
 }
 
+// chartFromAssets finds a Helm chart in the assets at location, creates a chart.Chart struct from
+// the chart and returns a pointer to it.
+func chartFromAssets(location string) (*chart.Chart, error) {
+	tmpDir, err := ioutil.TempDir("", "lokoctl-chart-")
+	if err != nil {
+		return nil, errors.Wrap(err, "creating temporary dir")
+	}
+	defer os.RemoveAll(tmpDir)
+
+	// Rendered files could contain secret data, only allow the
+	// current user but not others
```

Comment:
```suggestion
	// current user but not others.
```

Reply:
Yes, this was copy-pasted. I've fixed this comment in the meantime.